File: 프로그래머스/0/120583. 중복된 숫자 개수/중복된 숫자 개수.py
def solution(array, n):
    answer = 0

    # 풀이 1 - 반복문을 사용하여 카운트
    # for a in array:
    #     if a % n == 0:
    #         answer += 1

    # 풀이 2 - 리스트의 count() 함수 사용
    answer = array.count(n)

    return answer

print(solution([1, 1, 2, 3, 4, 5], 1))  # 2
print(solution([0, 2, 3, 4], 1))        # 0

File: 프로그래머스/0/120583. 중복된 숫자 개수/중복된 숫자 개수.go
// 배열에서 n이 등장하는 횟수를 반환하는 Go 함수
func solution(array []int, n int) int {
	answer := 0
	for i := 0; i < len(array); i++ {
		if array[i] == n {
			answer++
		}
	}
	return answer
}
